### DISCUSSION TITLE
Trying to show "Disabled" in 3.5e system, but always get "Staggered".

### DISCUSSION THREAD

**Discussion by Oryan77** [mclemente/healthEstimate#230] [Q&A]
I'm using the 3.5e system and the default D35E.CondStaggered table works fine when nonlethal damage equals current hitpoints. `					game.settings.get("healthEstimate", "PF1.showExtra") &&
					(totalHp === 0 ||
						(hp.nonlethal > 0 && totalHp == hp.nonlethal) ||
						Array.from(token.actor.effects.values()).some((x) => x.label === game.i18n.localize("D35E.CondStaggered")))`

However, it also adds the Staggered label when an actor reaches 0 hp without any nonlethal damage.

I'm trying to set Health Estimate to display "Disabled" instead when a PC actor reaches exactly 0 HP, but I can't figure out how to set this. I tried the JS Rule `actor.effects.find((effect) => effect.label === "Disabled")` but it doesn't work. "Disabled" is a default status effect condition, so maybe I'm doing something wrong because of that?

**Comment by mclemente** [comment]
> it also adds the Staggered label when an actor reaches 0 hp without any nonlethal damage.

Try this for the Staggered rule:
```js
game.settings.get("healthEstimate", "PF1.showExtra") && ((hp.nonlethal > 0 && totalHp == hp.nonlethal) || Array.from(token.actor.effects.values()).some((x) => x.label === game.i18n.localize("D35E.Staggered")))
```

And then this for Disabled rule:
```js
game.settings.get("healthEstimate", "PF1.showExtra") && Array.from(token.actor.effects.values()).some((x) => x.label === game.i18n.localize("D35E.Disabled"))
```

**Comment by mclemente** [comment]
Actually, if you haven't changed much on the Estimations (e.g. no new estimatives or tabs), update the module and then reset the settings.

Otherwise, try these (in order):

Disabled `game.settings.get(\"healthEstimate\", \"PF1.showExtra\") && (totalHp === 0 || Array.from(token.actor.effects.values()).some((x) => x.name === game.i18n.localize(\"D35E.Disabled\")))`.

Staggered: `game.settings.get(\"healthEstimate\", \"PF1.showExtra\") && (hp.nonlethal > 0 && totalHp == hp.nonlethal) || Array.from(token.actor.effects.values()).some((x) => x.name === game.i18n.localize(\"D35E.Staggered\"))`

Unconscious: `game.settings.get(\"healthEstimate\", \"PF1.showExtra\") && (hp.nonlethal > totalHp || Array.from(token.actor.effects.values()).some((x) => x.name === game.i18n.localize(\"D35E.Unconscious\")))`
